[2022-03-20 14:15:51,045] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-01-09T00:00:00+00:00 [queued]>
[2022-03-20 14:15:51,060] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-01-09T00:00:00+00:00 [queued]>
[2022-03-20 14:15:51,061] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 14:15:51,061] {taskinstance.py:1068} INFO - Starting attempt 2 of 2
[2022-03-20 14:15:51,061] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 14:15:51,098] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): processing_user> on 2020-01-09T00:00:00+00:00
[2022-03-20 14:15:51,104] {standard_task_runner.py:52} INFO - Started process 105722 to run task
[2022-03-20 14:15:51,121] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'processing_user', '2020-01-09T00:00:00+00:00', '--job-id', '1013', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpirw1zyoy', '--error-file', '/tmp/tmp0rnwxn32']
[2022-03-20 14:15:51,122] {standard_task_runner.py:77} INFO - Job 1013: Subtask processing_user
[2022-03-20 14:15:51,362] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.processing_user 2020-01-09T00:00:00+00:00 [running]> on host airflowvm
[2022-03-20 14:15:51,505] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2020-01-09T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-09T00:00:00+00:00
[2022-03-20 14:15:51,544] {python.py:151} INFO - Done. Returned value was: None
[2022-03-20 14:15:51,557] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=processing_user, execution_date=20200109T000000, start_date=20220320T141551, end_date=20220320T141551
[2022-03-20 14:15:51,719] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-20 14:15:51,765] {local_task_job.py:151} INFO - Task exited with return code 0
